webapp/src/pages/EditChatroomMemberPage/EditChatroomMemberPage.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  AvatarPageTemplate,
+  ButtonVariant,
+  IconVariant,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { useAuth } from '../../shared/hooks/UseAuth';
+import { AVATAR_EP_URL } from '../../shared/urls';
+
+export default function EditChatroomMemberPage() {
+  const { authUser, logout, isLoading: isAuthUserLoading } = useAuth();
+  return (
+    <AvatarPageTemplate
+      isLoading={isAuthUserLoading}
+      headerStatusVariant="online"
+      title={authUser?.username ?? ''}
+      avatarProps={{
+        url: `${AVATAR_EP_URL}/${authUser?.avatarId}`,
+        caption: 'level 4',
+        XCoordinate: authUser?.avatarX,
+        YCoordinate: authUser?.avatarY,
+      }}
+      button={{
+        variant: ButtonVariant.WARNING,
+        iconVariant: IconVariant.LOGOUT,
+        onClick: logout,
+        children: 'Logout',
+      }}
+    >
+      <>
+        <Text
```

Comment:
This is removed and improved in the followup [335](https://github.com/42AGV/ft_transcendence/pull/335)